canonicalwebteam/templates/image_template.html

```diff
@@ -1,12 +1,12 @@
 <img
   {% if hi_def %}
-   data-srcset="https://res.cloudinary.com/canonical/image/fetch/{{ hi_def_cloudinary_options }}/{{ url }} 2x"
+   {% if lazy != False %}data-{% endif %}srcset="https://res.cloudinary.com/canonical/image/fetch/{{ hi_def_cloudinary_options }}/{{ url }} 2x"
```

Comment:
This is kind of like a double negative here :)

maybe `if lazy == True` ?

Typically people prefer truthy checks `if x == True` in general

Reply:
or just `{% if lazy %}` I guess?